Userland/Applications/Browser/main.cpp

```diff
@@ -70,6 +82,17 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     Browser::g_home_url = Config::read_string("Browser", "Preferences", "Home", "file:///res/html/misc/welcome.html");
     Browser::g_search_engine = Config::read_string("Browser", "Preferences", "SearchEngine", {});
 
+    Browser::g_filetype_html_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/filetype-html.png"));
+    Browser::g_filetype_text_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/filetype-text.png"));
+    Browser::g_filetype_javascript_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/filetype-javascript.png"));
+    Browser::g_bookmark_contour_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/bookmark-contour.png"));
+    Browser::g_bookmark_filled_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/bookmark-filled.png"));
+    Browser::g_inspector_object_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/inspector-object.png"));
+    Browser::g_find_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/find.png"));
+    Browser::g_color_chooser_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/color-chooser.png"));
+    Browser::g_delete_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/delete.png"));
+    Browser::g_new_tab_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/new-tab.png"));
+
```

Comment:
Interesting idea to go about resource initialization - just initialize it all at startup! I'm not sure how I feel about all the globals, but it kind of makes sense to do things this way.